web-platform-when-why/index.html

```diff
@@ -0,0 +1,423 @@
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+    <link rel="stylesheet" href="css/reveal.min.css">
+    <link rel="stylesheet" href="css/theme/firefoxos.css" id="theme">
+
+    <!-- For syntax highlighting -->
+    <link rel="stylesheet" href="lib/css/zenburn.css">
+
+    <!-- If the query includes 'print-pdf', use the PDF print sheet -->
+    <script>
+     if( window.location.search.match( /print-pdf/gi ) ) {
+       var link = document.createElement( 'link' );
+       link.rel = 'stylesheet';
+       link.type = 'text/css';
+       link.href = 'css/print/pdf.css';
+       document.getElementsByTagName( 'head' )[0].appendChild( link );
+     }
+    </script>
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <div class="reveal">
+
+      <div class="slides">
+        <section>
+          <h1>The Right Choice</h1>
+          <h2>When is the Web the right answer &mdash;and why?</h2>
+          <aside class="notes">
+            Thank you for coming to The Right Choice. In this session I'm going to tell
+            you about the Web a platform, what are the benefits from Web technologies and
+            when to choose it.
+          </aside>
+        </section>
+
+        <section>
+          <h2>Index</h2>
+          <ol>
+            <li>The Web as a platform</li>
+            <li>So, what's stopping the Web?</li>
+            <li>Progressive Web Applications</li>
+            <li>Successful examples</li>
+            <li>Resources and credits</li>
+          </ol>
+          <aside class="notes">
+            This is the agenda for the talk. Let me convince you about why you should love
+            the Web, then I'll explain what are the main factors preventing the Web from
+            being a competitive platform to native solutions. To follow, let's see how
+            modern technologies will bring the native application experience to the Web
+            applications and finally, let's review some successful examples, resources and
+            the proper credits.
+          </aside>
+        </section>
+
+        <section>
+          <img alt="XKCD's Installing" src="./imgs/installing.png"/>
+          <aside class="notes">
+            I tend to start my talks with a quotation but this time, I'm quoting XKCD in
+            its comic strip "Installing". It's a very sharp observation to realize the
+            Web is always there, with no installation process, just one link away.
+          </aside>
+        </section>
+
+        <section>
+          <section>
+            <h2>Web vs Native</h2>
+            <aside class="notes">
+              Out of the agenda, let me start with a few words about Web vs Native wars.
```

Comment:
*out of* suggests that you're actually going to say something from the *agenda*. "As a side comment" or "I know it's early to deviate from my own agenda, but..." would be cool.